packages/snap-preact-components/src/components/Molecules/RadioList/RadioList.tsx

```diff
@@ -165,10 +141,10 @@ export interface RadioListProps extends ComponentProps {
 type option = {
```

Comment:
To ensure future type compatibility, should all of these components use a shared type instead of redefining it?